Mathlib/Tactic/CategoryTheory/Monoidal.lean

```diff
@@ -71,12 +65,12 @@ structure Context where
   C : Expr
 
 /-- Populate a `context` object for evaluating `e`. -/
-def mkContext (e : Expr) : MetaM Context := do
+def mkContext (e : Expr) : MetaM (Option Context) := do
```

Comment:
```suggestion
def mkContext? (e : Expr) : MetaM (Option Context) := do
```

I believe generally defs in `MetaM (Option ...)` should have a `?` at the end of their name.